global.json

```diff
@@ -5,7 +5,7 @@
     "rollForward": "major"
   },
   "tools": {
-    "dotnet": "5.0.100"
+    "dotnet": "6.0.100-preview.2.21118.7"
```

Comment:
?? Is this intended?

Reply:
Yes, see the commit description :) It is necessary as the IL.SDK import sequence was wrong and causes problems.

Reply:
More precisely, we need this fix: https://github.com/dotnet/sdk/commit/309c55fbf841285acb5772090f65de3916dee363